packages/expo-asset/src/PlatformUtils.ts

```diff
@@ -38,20 +38,19 @@ export const manifestBaseUrl = Constants.experienceUrl
   ? getManifestBaseUrl(Constants.experienceUrl)
   : null;
 
-// TODO: how should this behave in bare app with updates? re: hashAssetFiles
 export async function downloadAsync(uri, hash, type, name): Promise<string> {
-  if (IS_MANAGED_ENV) {
-    return _downloadAsyncManagedEnv(uri, hash, type, name);
+  if (hash !== null) {
```

Comment:
this is not sufficient to determine if `hashAssetFiles` is being used. metro always includes a hash for the asset, but it isn't the md5 hash of the file real asset (there can be multiple asset sources for a single id, eg: @2x, @3x).